src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -5,6 +5,7 @@
 
 public class HibernateUtil {
     private static final SessionFactory sessionFactory = initSessionFactory();
```

Comment:
Consider adding a shutdown method to close the `SessionFactory` when the application is stopped. This can help in releasing resources properly.